app/models/katello/glue/candlepin/pool.rb

```diff
@@ -57,7 +85,9 @@ def stacking_subscription(org_label, stacking_id)
 
     module InstanceMethods
       def import_lazy_attributes
-        json = self.backend_data
+        json = self.class.candlepin_data(self.cp_id, true)
+
+        return {} if json.blank?
```

Comment:
Seems like this is actually a hash all along?  If so, kind of confusing to call it `json`.
If it's actually a JSON string, should return `{}` as a JSON string if `json.blank?`.

Reply:
Agree, but due to the sensitive nature of this change I don't want to update all variable references in this method and thereby increase the necessary testing scope